javascript/javascript3/week1/cool_api.js

```diff
@@ -0,0 +1,87 @@
+const input = document.getElementById("city");
+const mainDiv = document.getElementById("weather-data");
+let userInput = "";
+function onChange(event) {
+    userInput = event.target.value;
+}
+input.addEventListener("input", onChange);
+
+function showCurrentWeather() {
+    if(userInput === "") {
+        alert("Please type your city name first...");
+    } else {
+        document.getElementById("input-div").style.display = "none";
```

Comment:
Very long code with repetitive bits. It's hard to read and understand. Your functions should generally not be more than 1 page long, and if they are, it's a sign that you should split it up into more functions.

Reply:
Yes, got that. Will try to break that into smaller chunks.